src/cmd7.cc

```diff
@@ -3555,7 +3554,7 @@ void do_cmd_unbeliever()
 	/* Select what to do */
 	while (TRUE)
 	{
-		if (!get_com("Disrupt [C]ontinuum or [D]etect Traps", &ch))
+		if (!get_com("Disrupt [C]ontinuum", &ch))
```

Comment:
I think we should just ditch the whole menu here and promote the disruption of the "Anti-magic" skill... but that should probably be done as a follow-up.